ocaml/idl/datamodel.ml

```diff
@@ -4545,6 +4545,13 @@ module PGPU = struct
           param_release = vgpu_tech_preview_release;
           param_default = None;
         };
+        {
```

Comment:
Let's not add this to the datamodel. As we only need this additional list for internal computations, it should not be part of the external API.